plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt

```diff
@@ -430,10 +431,10 @@ class FeatureDevController(
             }
         } catch (err: Exception) {
             logger.warn(err) { "Encountered ${err.message} for tabId: $tabId" }
-            if (err is ContentLengthError) {
+            if (err is RepoSizeLimitError || err is ContentLengthError) {
                 messenger.sendError(
                     tabId = tabId,
-                    errMessage = err.message,
+                    errMessage = err.message!!,
```

Comment:
I don't know why i need to use `!!` in this case, since the 2 errors has message. But if i removed it the compiler complain saying that the message can be optional and it's expected to be required

Reply:
Most probably the type is inferring from RepoSizeLimitError has a possible undefined message. And I guess it is becuase it just extends RuntimeException(). Maybe the root cause is on that.

Reply:
does it work in a switch statement?

Reply:
But both of them are extended from `RuntimeException()`, why it's working fine with both of them separate in different if conditions?

Anyway if you know how to fix it, let's huddle

Reply:
> does it work in a switch statement?

Tried didn't work :(